client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/index.js

```diff
@@ -45,6 +48,17 @@ const customsSummary = ( errors, translate ) => {
 const CustomsStep = props => {
 	const { siteId, orderId, errors, expanded, translate, isSubmitted, packageDescriptions } = props;
 
+	const [ isEuCustomsPopoverVisible, setIsEuCustomsPopoverVisible ] = useState(
+		Object.keys( window.wcsPluginData.notices ).includes( 'wcs-usps-eu-custom-description-mar-01-2023' )
```

Comment:
The extension's PHP code now uses `wp_localize_script()` to expose a list of undismissed notices to the front-end.

This PR then hijacks that list of notices and if the EU customs notice hasn't yet been dismissed, it will exclude that notice from the classic rendering (in `woocommerce-services.php`) and instead display it as a popover.